packages/cli/src/commands/new/project.ts

```diff
@@ -73,20 +79,14 @@ export default class Project extends Command {
   }
 }
 
-const run = async (flags: Partial<ProjectInitializerConfig>, boosterVersion: string): Promise<void> => {
-  const boostScript = Script.init(`boost ${Brand.energize('new')} 🚧`, parseConfig(new Prompter(), flags, boosterVersion))
+const run = async (flags: Partial<ProjectInitializerConfig>, boosterVersion: string): Promise<void> =>
+  Script.init(`boost ${Brand.energize('new')} 🚧`, parseConfig(new Prompter(), flags, boosterVersion))
     .step('Creating project root', generateRootDirectory)
     .step('Generating config files', generateConfigFiles)
-
-  if (flags.skipInstall) {
-    return boostScript.info('Project generated!').done()
-  }
-
-  return boostScript
-    .step('Installing dependencies', installDependencies)
+    .optionalStep(Boolean(flags.skipInstall), 'Installing dependencies', installDependencies)
```

Comment:
From [mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean) "The value passed as the first parameter is converted to a boolean value, if necessary."

It simply converts value to boolean, because of `Partial<ProjectInitializerConfig>` every flag value is optional and may be "undefined", eslint tells that value may be `boolean | undefined` and doesn't like that, so using `Boolean()` I convert value to true/or false, just to be sure.

Reply:
Ah, I see :) Awesome :D